pg_mooncake/benchmark.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+
+sudo apt-get update
+sudo apt-get install -y docker.io
+sudo apt-get install -y postgresql-client
+
+wget --no-verbose --continue https://datasets.clickhouse.com/hits_compatible/athena/hits.parquet
+sudo docker run -d --name pg_mooncake -p 5432:5432 -e POSTGRES_HOST_AUTH_METHOD=trust - -v ./hits.parquet:/tmp/hits.parquet mooncakelabs/pg_mooncake
```

Comment:
L. 9 gives me `docker: invalid reference format.`.

What worked for me was

```
docker run -d --name pg_mooncake -p 5432:5432 -e POSTGRES_HOST_AUTH_METHOD=trust -v ./hits.parquet:/tmp/hits.parquet -- mooncakelabs/pg_mooncake
```

Kindly double-check.

Reply:
+1 to get rid of `sudo` parts and make all preparations in the running container.

Let's assume that docker is installed on a host. And we should not stick to the Debian ecosystem.